content/docs/configuration/content.md

```diff
@@ -6,58 +6,214 @@ title = "Configuration"
 
 [sidebar]
   enable = true
+  sticky = true
 +++
 
-**Redirect all requests to "www"-subdomain:**  
-*example.com -> www.example.com*
+# Redirect Types
+
+---
+
+## Supported redirect types:
+
+| Type Name | Type Description                                            |
```

Comment:
This is mostly an issue with Syna not making markdown tables look pretty yet. You can bug @mpourismaiel about it ;)